RELEASES.md

```diff
@@ -2,6 +2,9 @@
 
 ## Pending Release
 
+- Added `eth_suggestPriceOptions` API to suggest gas prices (slow, normal, fast) based on the current network conditions
+- Added `"po-slow-fee-percentage"`, `"po-fast-fee-percentage"`, `"po-max-base-fee"`, and `"po-max-tip"` fields to chainConfig to configure the new `eth_suggestPriceOptions` API
```

Comment:
nit: let's avoid the term chainConfig, initially it confused me. I think usually we refer to these as configuration options or flags.